### PR TITLE
add Interaction Media Features

### PR DESCRIPTION
**Pull request by cvrebert**
See http://dev.w3.org/csswg/mediaqueries-4/#mf-interaction

### CODE REVIEW

File: posts/interaction-media-features.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+feature: Interaction media features (hover, pointer, any-hover, any-pointer)
```
Comment (arthurvr) [inline posts/interaction-media-features.md]: Is the last part of the title (the piece in parenthesis) needed? This way the title appears to be a little long.

Comment (cvrebert) [inline posts/interaction-media-features.md]: I was copying the style of the "Viewport units (vh, vw, vmin, vmax)" entry.
Perhaps the `any-*`s should be dropped? Or maybe I should drop "Interaction media features"?

Comment (arthurvr) [inline posts/interaction-media-features.md]: I think we can just go with `Interaction media features` in here. No real reason to sum them up.

File: posts/interaction-media-features.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+status: caution
+tags: polyfill
+kind: css
+polyfillurls: [mq4-hover-hover-shim](https://github.com/cvrebert/mq4-hover-hover-shim)
```
Comment (arthurvr) [inline posts/interaction-media-features.md]: Are there any good candidates for `moreurl`'s you can think off? (see other posts for examples)

Comment (cvrebert) [inline posts/interaction-media-features.md]: Maybe http://www.jordanm.co.uk/post/66660448615/potential-use-cases-for-script-hover-and-pointer ?

File: posts/interaction-media-features.md (hunk 1 above)
Comment (cvrebert) [inline posts/interaction-media-features.md]: Okay, changed.

**Comment by cvrebert** [comment]
@arthurvr Added the `moreurl` I mentioned in my comment.

**Comment by arthurvr** [comment]
Okay, thanks @cvrebert! In general this post LGTM, but I'm going to let it open a few days for others to review it.

**Comment by arthurvr** [comment]
Landed. Thanks once again, @cvrebert!

**Pull request closed without merging**